src/main/java/org/springframework/samples/petclinic/vet/Vet.java

```diff
@@ -15,11 +15,7 @@
  */
 package org.springframework.samples.petclinic.vet;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
We never use "*" imports in Spring projects. Probably best to maintain that rule here.